tests/CTestCustom.ctest.in

```diff
@@ -5,19 +5,5 @@
 SET(RUN_ALL_TESTS @ASPECT_RUN_ALL_TESTS@)
 
 IF(NOT RUN_ALL_TESTS)
-  SET(CTEST_CUSTOM_POST_TEST
-      "echo ''"
-      "echo ''"
-      "echo '--------------------------------------------------------------'"
-      "echo 'Completed running the minimal testsuite. Please check the test'"
-      "echo 'results above. To run the complete testsuite, reconfigure your'"
-      "echo 'project with the option ASPECT_RUN_ALL_TESTS=ON or by running'"
-      "echo '    make setup_tests'"
-      "echo 'Be aware that installing the 'numdiff' tools is strongly'"
-      "echo 'recommended before running the full testsuite. Even then, the'"
-      "echo 'output is likely different on your machine due to configuration'"
-      "echo 'differences, so do not expect all tests to succeed on your'"
-      "echo 'machine.'"
-      "echo '--------------------------------------------------------------'"
-    )
+  SET(CTEST_CUSTOM_POST_TEST "./print_test_info.sh")
```

Comment:
I don't get it. Both form of this execute commands. What's the difference? Do you get execution errors? If so, does it help if you terminate each line with a semicolon?